src/content/learn/BestPractice-ServingOverHTTP.md

```diff
@@ -87,4 +80,4 @@ If you are using NodeJS, we recommend looking at the [list of server implementat
 
 ## Draft Transport Specification
 
-A detailed [HTTP & websockets transport specification](https://github.com/graphql/graphql-over-http) is in development. Though it is not yet finalized, these draft specifications act as a single source of truth for GraphQL client & library maintainers, detailing how to expose and consume a GraphQL API using an HTTP transport. Unlike the language specification, adherence is not mandatory, but most implementations are moving towards these standards to maximize interoperability.
\ No newline at end of file
+A detailed [HTTP & websockets transport specification](https://github.com/graphql/graphql-over-http) is in development. Though it is not yet finalized, these draft specifications act as a single source of truth for GraphQL client & library maintainers, detailing how to expose and consume a GraphQL API using an HTTP transport. Unlike the language specification, adherence is not mandatory, but most implementations are moving towards these standards to maximize interoperability.
```

Comment:
Also SSE transport.

---

Why's adherence not mandatory? In order to be GQL over HTTP compliant - it _is_ mandatory. I'd recommend rephrasing or just removing this sentence altogether.

Reply:
It's not mandatory to be GraphQL compliant; you can be GraphQL compliant without being GraphQL-over-HTTP compliant. For example, persisted operations (over HTTP) still comply with the GraphQL spec but don't (currently) comply with GraphQL-over-HTTP.

Also I'd replace "HTTP & websockets & SSE" with just "HTTP" - all of these things are part of HTTP.

Reply:
FWIW I didn't actually make edits to this section (looks like my editor added a missing newline). But agree with @benjie 's suggestion here and will simplify this to just "HTTP"